packages/platform/platform-mcp/test/mcp.integration.spec.ts

```
@@ -0,0 +1,178 @@
+  let request: SuperTest.Agent;
+
+  beforeEach(
+    utils.bootstrap({
+      mcp: {
+        path: "/mcp"
+      }
+    })
+  );
+  beforeEach(() => {
+    request = SuperTest(PlatformTest.callback());
+  });
```

Biome (2.3.13)

[error] Duplicate beforeEach hook found.

Remove this duplicate hook or consolidate the logic into a single hook.

(lint/suspicious/noDuplicateTestHooks)